src/main/java/org/zaproxy/zap/extension/jwt/JWTExtension.java

```diff
@@ -31,25 +48,77 @@
 public class JWTExtension extends ExtensionAdaptor {
 
     protected static final Logger LOGGER = Logger.getLogger(JWTExtension.class);
+    private static final List<Class<? extends Extension>> DEPENDENCIES;
+
+    static {
+        List<Class<? extends Extension>> dependencies = new ArrayList<>(1);
+        dependencies.add(ExtensionHttpFuzzer.class);
+        DEPENDENCIES = Collections.unmodifiableList(dependencies);
+    }
+
+    private HttpFuzzerHandler httpFuzzerHandler;
+    private JWTMessageLocationReplacerFactory jwtMessageLocationReplacerFactory;
+
+    @Override
+    public List<Class<? extends Extension>> getDependencies() {
+        return DEPENDENCIES;
+    }
 
     @Override
     public String getAuthor() {
         return "KSASAN preetkaran20@gmail.com";
     }
 
+    @Override
+    public void init() {
+        JWTI18n.init();
+        jwtMessageLocationReplacerFactory = new JWTMessageLocationReplacerFactory();
+        MessageLocationReplacers.getInstance()
+                .addReplacer(HttpMessage.class, jwtMessageLocationReplacerFactory);
+    }
+
+    @Override
+    public void initView(ViewDelegate view) {
+        super.initView(view);
+
+        HttpPanelManager panelManager = HttpPanelManager.getInstance();
```

Comment:
The Fuzzer add-on now allows to add views to just the the Fuzzer dialogue, through `ExtensionFuzz#getClientMessagePanelManager` and `#getServerMessagePanelManager()`.

Reply:
@thc202, I am not sure which one to use also i tried using clientMessagePanelManager and i am getting null pointer. I am not sure why. I am thinking that clientMessagePanelManager gets initialized in initview and hence may be that is causing issue.

Reply:
It would be the "client" which in HTTP maps to "request". That's right, that would have to be done later in the hook method (if there's view), by then the view of the fuzz extensions is already initialised.